docs/modules/ROOT/pages/vshn-managed/postgresql/backup.adoc

```diff
@@ -0,0 +1,78 @@
+= How to manage backups
```

Comment:
```suggestion
= How to work with backups
```

As a user, I would be very much interested how to restore a backup, this is completely missing in this page. Why?

Reply:
We already have a restore page. Now the question is, do we really need this page at all?

Reply:
How about combining them into a Backup/Restore page?

Reply:
I talked with @Kidswiss and we agreed that it would be best to have `list` and `get` operation directly in the [restore page](https://github.com/appuio/appcat-docs/pull/34) with as less technicality as possible. I will close this PR as it's too technical and does not make sense to have it presented to the end user in this format.